docs/book/src/usage/ci.md

```diff
@@ -0,0 +1,125 @@
+# CI
+
+Marker's explicit goal is to make it easy to use on CI. This document describes the provided CI tooling and demonstrates example templates.
+
+<!-- toc -->
+
+## GitHub Action
+
+Marker provides a GitHub Action that downloads the pre-compiled binaries and runs `cargo marker`.
+
+<!-- region replace-version stable -->
+
+```yml
+- uses: rust-marker/marker@v0.2
+```
+
+### Git tags
+
+The git tag used for the GitHub Action selects the version of Marker that will be installed. There are several flavours of git tags:
+
+- Sliding `v0.2` *(recommended)*. Use this to get automatic patch updates.
+- Fixed `v0.2.1`. Use this if you want to pin a specific version if there is a regression in a new patch version. If there is one, please create a [new issue], patch versions must never break anything!
+
+<!-- endregion replace-version stable -->
+
+> ⚠️ Different `v0.{minor}` versions contain breaking changes. There is `v0` git tag, but because there are breaking changes on each minor release it doesn't make sense to use it. However once Marker reaches `1.0` minor versions will be compatible and `v1` tag will be stable.
+
+### Inputs
+
+All inputs are optional, they only allow tweaking the default behavior.
+
+| Name         | Description                                               | Type      | Default |
+|--------------|-----------------------------------------------------------|-----------|---------|
+| `install-only` | Only install Marker and don't run `cargo marker` command. | `boolean` | `false` |
+
+### Example workflows
+
+#### Basic usage
+
+Checkout the repository code, install the toolchain, Marker, and start linting.
+
+```yml
+jobs:
+    rust-marker-lints:
+        runs-on: ubuntu-latest
```

Comment:
Warnings will cause rustc and Marker to still exit with 0, for CI we would want to deny warnings:

```suggestion
        runs-on: ubuntu-latest
        env:
            RUSTFLAGS: --deny warnings
```

The `env` field could also be specified above the `jobs` section, like it's done in Marker's CI, but having it here should make copy-and-pasta :spaghetti: easier :)

Same for the example below.

Reply:
Yeah, totally forgot about this one, thanks